### ISSUE TITLE
Cannot detect HP F4580 scanner

### ISSUE THREAD

**Issue by eheintzmann** [flathub/org.gnome.SimpleScan#22]
The flatpak of Simple Scan 46 does not detect the scanner of my HP Deskjet F4580.
But if it is installed from the distribution, it works fine.

The logs :
```bash
$ flatpak run org.gnome.SimpleScan --debug
[+0,00s] DEBUG: simple-scan.vala:2016: Starting simple-scan 46.0, PID=2
[+0,01s] DEBUG: Using cross-namespace EXTERNAL authentication (this will deadlock if server is GDBus < 2.73.3)
[+0,03s] CRITICAL: g_application_set_application_id: assertion 'application_id == NULL || g_application_id_is_valid (application_id)' failed
[+0,03s] DEBUG: org.gtk.vfs.MountTracker.listMountableInfo call failed: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: org.freedesktop.DBus.Error.ServiceUnknown (g-dbus-error-quark, 2)

[+0,03s] DEBUG: _g_io_module_get_default: Found default implementation local (GLocalVfs) for ‘gio-vfs’
[+0,15s] DEBUG: _g_io_module_get_default: Found default implementation keyfile (GKeyfileSettingsBackend) for ‘gsettings-backend’
[+0,18s] DEBUG: app-window.vala:1436: Loading state from /home/eric/.var/app/org.gnome.SimpleScan/config/simple-scan/state
[+0,18s] DEBUG: app-window.vala:1415: Restoring window to 600x480 pixels
[+0,24s] DEBUG: scanner.vala:1619: sane_init () -> SANE_STATUS_GOOD
[+0,24s] DEBUG: scanner.vala:1625: SANE version 1.3.0
[+0,24s] DEBUG: scanner.vala:1686: Requesting redetection of scan devices
[+0,24s] DEBUG: scanner.vala:863: Processing request
[+0,39s] DEBUG: app-window.vala:1512: Saving state to /home/eric/.var/app/org.gnome.SimpleScan/config/simple-scan/state
[+5,14s] DEBUG: scanner.vala:348: sane_get_devices () -> SANE_STATUS_GOOD
[+5,15s] DEBUG: scanner.vala:1686: Requesting redetection of scan devices
[+5,15s] DEBUG: scanner.vala:863: Processing request
[+5,26s] DEBUG: app-window.vala:1512: Saving state to /home/eric/.var/app/org.gnome.SimpleScan/config/simple-scan/state
[+7,34s] DEBUG: scanner.vala:348: sane_get_devices () -> SANE_STATUS_GOOD
```

The printer is connected over USB
```bash
flatpak run --command=sane-find-scanner org.gnome.SimpleScan | grep HP
found possible USB scanner (vendor=0x03f0 [HP], product=0x8c11 [Deskjet F4500 series]) at libusb:003:006
```

(Probably related to #9 and #19)

Thanks

**Comment by hfiguiere** [comment]
Like already filed #9

**Comment by eheintzmann** [comment]
Fixed by #23
